src/patternfly/components/HelperText/helper-text.scss

```diff
@@ -16,6 +16,8 @@
   --#{$helper-text}__item-text--m-success--FontWeight: var(--pf-t--global--font--weight--body--bold);
   --#{$helper-text}__item-icon--m-error--Color: var(--pf-t--global--icon--color--status--danger--default);
   --#{$helper-text}__item-text--m-error--FontWeight: var(--pf-t--global--font--weight--body--bold);
+  --#{$helper-text}__item--m-error--TransitionDuration--Opacity: var(--pf-t--global--motion--duration--fade--default);
+  --#{$helper-text}__item--m-error--TransitionTimingFunction--Opacity: var(--pf-t--global--motion--timing-function--default);
```

Comment:
I think we've been doing lower case for the suffix `--opacity` - I'm fine either way though.

Reply:
I went with uppercase since, in this file, all the other variables with names used `--PropertyName`. But I noticed it was different in other files.